core/src/main/java/com/amazon/opendistroforelasticsearch/sql/expression/operator/predicate/UnaryPredicateOperator.java

```diff
@@ -74,6 +90,20 @@ private static FunctionResolver isNull() {
                 Collectors.toList()));
   }
 
+  private static FunctionResolver isNull() {
```

Comment:
From my point of view, this method should be identical to the `is_Null()` method above. The setting of `UNKNOWN` type is not necessary